### PR TITLE
CSS: Import a Google Font .md

### PR DESCRIPTION
**Pull request by bknuertey**
Replace this sentence with a brief description of the awesome changes your PR has. 😊

---

## ✅️ By submitting this PR, I have verified the following

- [x] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [x] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [x] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

File: src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md [hunk 1]
```diff
@@ -3,8 +3,21 @@ title: Import a Google Font
 ---
 ## Import a Google Font
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+````CSS
```
Comment (Asjas) [inline src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md]: Please use a `html` codeblock for the `link` declaration.

File: src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md [hunk 2]
```diff
@@ -3,8 +3,21 @@ title: Import a Google Font
...
+ <link href="https://fonts.googleapis.com/css?family=Lobster" rel="stylesheet" type="text/css">
+
+<style>
```
Comment (Asjas) [inline src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md]: Use a `css` codeblock for the `css` declaration.

File: src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md [hunk 3]
```diff
@@ -3,8 +3,21 @@ title: Import a Google Font
...
+  }
+  h2 {
+    font-family: Lobster;
+  }
+  p {
+    font-size: 16px;
+    font-family: monospace;
+  }
+</style>
+
```
Comment (Asjas) [inline src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md]: Add the missing syntax to close the codeblock.

File: src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md (hunk 1 above)
Comment (bknuertey) [reply src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md]: I will do so. Thanks

File: src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md (hunk 3 above)
Comment (bknuertey) [reply src/pages/certifications/responsive-web-design/basic-css/import-a-google-font/index.md]: Can you please check if i have done it right.

**Comment by Asjas** [comment]
Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :tada:

**Pull request merged**